presets/shared.yml

```diff
@@ -214,3 +215,9 @@ connectionPoolSize: 10
 maxSubscriptions: 300
 restExtensions: 'accountLink, aggregate, lockHash, lockSecret, mosaic, metadata, multisig, namespace, receipts, restrictions, transfer, cmc'
 restDeploymentTool: symbol-bootstrap
+restProtocol: HTTP
```

Comment:
`restProtocol` is the protocol of the REST compose service. We don't know which option the user will take to enable HTTPS at this point. If the let's encrypt option is selected, `restProtocol` would still be HTTP because the HTTPS protocol will be covered by the reverse-proxy(https-portal).